lib/widgets/breez_avatar.dart

```diff
@@ -54,7 +57,11 @@ class BreezAvatar extends StatelessWidget {
         return _VendorAvatar(radius, avatarURL);
       }
 
-      return _NetworkImageAvatar(avatarURL, radius);
+      if (Uri.parse(avatarURL).scheme.startsWith("http")) {
```

Comment:
This will throw a FormatException if it is not a valid URI (or file path).

Reply:
```suggestion
      var uri = Uri.tryParse(avatarURL);
      if (uri != null && uri.scheme.startsWith("http")) {
        return _NetworkImageAvatar(avatarURL, radius);
      }
```

Reply:
Change looks good.